src/UNIV2LPOracle.sol

```diff
@@ -210,8 +211,8 @@ contract UNIV2LPOracle {
     }
 
     function step(uint256 _hop) external auth {
-        require(_hop <= uint16(-1), "UNIV2LPOracle/invalid-hop");
-        hop = uint16(_hop);
+        require(_hop <= uint24(-1), "UNIV2LPOracle/invalid-hop");
+        hop = uint24(_hop);
         emit Step(hop);
```

Comment:
Is this another SLOAD? Maybe you can just `emit Step(_hop);` here.

Reply:
Hmmmm, quite possibly...I was only focused on optimizing the `pass`+`poke` hot path in this PR, but it wouldn't hurt to touch up some other places as well.

Reply:
Just tried it out--it doesn't seem to eliminate an `SLOAD`, but it does save 41 gas (note that it has to do at least one `SLOAD` anyway to construct a packed word for writing, so it seems that Solidity is smart enough to reuse the written value).

I'll go ahead and make the change since it's a small savings.